userspace/libsinsp/sinsp_ppm_sc.cpp

```diff
@@ -238,6 +251,57 @@ std::unordered_set<uint32_t> sinsp::enforce_sys_ppm_sc_set(std::unordered_set<ui
 	return ppm_sc_set;
 }
 
+std::unordered_set<std::string> erase_io_syscalls_ppm_sc_codes(std::unordered_set<uint32_t>& ppm_sc_of_interest, std::unordered_set<uint32_t>& io_ppm_sc_set)
+{
+	std::unordered_set<uint32_t> cur_ppm_sc_set = ppm_sc_of_interest;
+	std::unordered_set<std::string> erased_io_syscalls_names = {};
+
+	for (const auto &ppm_sc_code : cur_ppm_sc_set)
+	{
+		if (io_ppm_sc_set.find(ppm_sc_code) != io_ppm_sc_set.end())
+		{
+			ppm_sc_of_interest.erase(ppm_sc_code);
+			erased_io_syscalls_names.insert(g_infotables.m_syscall_info_table[ppm_sc_code].name);
+		}
+	}
+	return erased_io_syscalls_names;
+}
+
+std::unordered_set<uint32_t> enforce_events(std::unordered_set<uint32_t> ppm_event_info_of_interest)
```

Comment:
Mmmh i think that non-syscalls events are always enabled, right?
Why do we need this?